modules.json

```diff
@@ -15,16 +15,6 @@
                         "git_sha": "65ad3e0b9a4099592e1102e92e10455dc661cf53",
                         "installed_by": ["modules"]
                     },
-                    "hisat2/build": {
-                        "branch": "master",
-                        "git_sha": "3f5420aa22e00bd030a2556dfdffc9e164ec0ec5",
-                        "installed_by": ["modules"]
-                    },
-                    "hisat2/extractsplicesites": {
-                        "branch": "master",
-                        "git_sha": "3f5420aa22e00bd030a2556dfdffc9e164ec0ec5",
-                        "installed_by": ["modules"]
-                    },
                     "multiqc": {
```

Comment:
Don't you want to keep that?

Reply:
Maybe... I'll reinstate in future PRs if so. Thanks!